server.js

```diff
@@ -55,6 +55,12 @@ app.prepare().then(() => {
         Location: req.url.replace('commands-reference', 'command-reference')
       })
       res.end()
+    } else if (pathname == '/doc/tutorial') {
+      // TMP: path /doc/tutorial -> /doc/tutorials/tutorial
+      res.writeHead(302, {
+        Location: req.url.replace('/doc/tutorial', '/doc/tutorials/tutorial')
```

Comment:
let's actually do tutorial -> tutorials? It should be even better since we have now more options, and we'll have more flexibility to remove the `tutorials/tutorial` if needed moving forward